engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorData.scala

```diff
@@ -19,17 +22,25 @@ final case class WorkflowExecutionDiff(executionStore: Map[JobKey, ExecutionStat
 
 case class WorkflowExecutionActorData(workflowDescriptor: EngineWorkflowDescriptor,
                                       executionStore: ExecutionStore,
-                                      outputStore: OutputStore) extends WdlLookup {
+                                      outputStore: OutputStore,
+                                      serviceRegistryActor: ActorRef) extends WdlLookup {
```

Comment:
Does this actorRef really need to be here in the FSM data? It feels a little awkward to carry this around.. perhaps can the stuff here related to Metadata be handled in the WorkflowExecutionActor ?